Common/GPU/OpenGL/thin3d_gl.cpp

```diff
@@ -569,6 +569,10 @@ OpenGLContext::OpenGLContext() {
 		caps_.fragmentShaderDepthWriteSupported = true;
 	}
 
+#ifndef USING_GLES2
+	caps_.logicOpSupported = true;
```

Comment:
Why not use `= gl_extensions.IsGLES`?  This would be the only cap enabled by that define.  I realize the func isn't available to link, so the ifdef sorta makes sense... but it'd be nice if GLES could still be used even without USING_GLES2.

-[Unknown]

Reply:
Yeah, that's better, dunno what I was thinking.